content/systems.md

```diff
@@ -30,27 +30,27 @@ The software configuration will consist of the following:
     - geany
     - kate
 - IDEs:
-    - IntelliJ Community
-        - Version: 2021.3.2
-        - Workspace is already set up
+    - Idea Community
+        - Version: 2022.3
+        - Workspace is already set up but needs to be opened from the welcome screen
```

Comment:
If this is not the default flow when opening IntelliJ, we shouldn't forget to mention this during the opening presentation. The default flow I have in mind, is:
- You double-click the IntelliJ icon
- The "Welcome" popup window appears
- You need to select the already-set-up workspace

Also, given that you did not write this for CLion and PyCharm, does that mean that those editor _do_ directly load into the already-set-up workspace on launch?

Reply:
Yes, indeed. CLion and PyCharm open the workspace directly.

Reply:
But it is easy to see what to click, the workspace is just listed in the welcome screen

Reply:
Exactly, just what I expected :slightly_smiling_face: Thanks for clarifying! :smile: